modules/ROOT/pages/rate-limiting-and-throttling-sla-based-policies.adoc

```diff
@@ -3,80 +3,132 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Rate Limiting policies based on a service level access (SLA) are client ID-based policies that use the client ID as a reference to impose limits on the number of requests that each application can make within a period of time. To use these policies you need to create at least one SLA tier to define request limits.
+:keywords: rate limiting SLA, api gateway, gateway, policy
 
-You can configure an SLA tier to require manual approval of requests from users who want to register an app to call your API.
 
-== Rate Limiting - SLA-Based Policy
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy name | Rate limiting SLA
+>s| Summary      | Monitors access to an API by defining a maximum amount of requests processed for a period of time based on SLAs
+>s| Category | Quality of Service
+>s| First Mule version available | v4.1.0
+.4+>.^s| Returned Status Codes
+|400 - Request is blocked because quota exceeded by WSDL APIs that use SOAP v1.2
+|401 - Request is blocked due to invalid client credentials (client ID or client secret) provided
+|429 - Quota exceeded, requests are blocked until the current window finishes
+|500 - Request is blocked because quota exceeded by WSDL APIs that use SOAP v1.1
```

Comment:
This status code is the error status code for SOAP v1.1, error can be a quota exceed OR an invalid client. The later error type is not explained here.

Reply:
Done! Thanks!